packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerVisual.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { from } from '@guardian/src-foundations/mq';
+import { ImageAttrs, ResponsiveImage } from '../../../shared/ResponsiveImage';
+import { Image } from '@sdc/shared/src/types';
+
+// ---- Component ---- //
+
+interface MomentTemplateBannerVisualProps {
+    settings: Image;
+}
+
+export function MomentTemplateBannerVisual({
+    settings,
+}: MomentTemplateBannerVisualProps): JSX.Element {
+    const baseImage: ImageAttrs = {
+        url: settings.mainUrl,
+        media: '',
+        alt: settings.altText,
+    };
+
+    const images: ImageAttrs[] = [];
+    if (settings.mobileUrl) {
+        images.push({ url: settings.mobileUrl, media: '(max-width: 739px)' });
+    }
+    if (settings.tabletUrl) {
+        images.push({ url: settings.tabletUrl, media: '(max-width: 979px)' });
+    }
+    if (settings.desktopUrl) {
+        images.push({ url: settings.desktopUrl, media: '(max-width: 1139px)' });
+    }
+    if (settings.leftColUrl) {
+        images.push({ url: settings.leftColUrl, media: '(max-width: 1299px)' });
+    }
+    if (settings.wideUrl) {
+        images.push({ url: settings.wideUrl, media: '' });
```

Comment:
I'm guessing this works fine so no need to change it now, but perhaps `media` should be optional?

Reply:
Yeah good shout 👍